src/EnergyPlus/BranchInputManager.cc

```diff
@@ -1518,7 +1518,7 @@ namespace BranchInputManager {
 		int NumNodes; // Number of Nodes from NodeInputManager
 
 		Branch( BCount ).Name = Alphas( 1 );
-		Branch( BCount ).MaxFlowRate = Numbers( 1 );
+//		Branch( BCount ).MaxFlowRate = Numbers( 1 );
```

Comment:
We mentioned this on one of our sizing calls and I can't remember what we resolved.  I think we decided that yes, we'd like to go ahead and get rid of it.  If you can add it here, that's cool, but if you need/want to put it in a separate, smaller branch of changes, that's fine with me too.